addons/pos_adyen/static/src/js/payment_adyen.js

```diff
@@ -211,6 +211,7 @@ var PaymentAdyen = PaymentInterface.extend({
                 self.remaining_polls--;
             }
 
+            // Need check if payment_identifier match
```

Comment:
We need verify if `notification[payment_identifier]` match with `payment_identifier` request to avoid issues with multi sessions